scala-generator/src/main/scala/models/Play2RouteGenerator.scala

```diff
@@ -108,6 +108,11 @@ private[models] case class Play2Route(
             case ScalaDatatype.Option(_) => None
             case p: ScalaPrimitive => Some("?= " + defaultForPrimitive(p, d))
           }
+        ).orElse(
+          param.datatype match {
+            case ScalaDatatype.Option(_) => Some("?= None")
```

Comment:
Not sure we want to provide a default for _all_ Options here (unsure if/what this might break). How. about we only provide the default for ScalaDatatype.Option(ScalaDatatype.List(_)) and ScalaDatatype.Option(ScalaDatatype.Map(_))

Reply:
i tested it with a Optional[Long] and there is no need to force the default to None, so as you say only keeping it to List and Map on the meantime,